### PR TITLE
Model task

### PR DESCRIPTION
**Pull request by palant-dev**
Change log (after the code review with Vasily):

- Simplified the entities in core data removing the relations (see Discord code channel more info)
- Generated classes for core data manually for better handling
- Removed the optional checkbox from the attributes model editor and removed the optionality to the stored properties inside the code
- Added model structs if we want to store first the data locally and then pass the data to core data:
- - Created PersonalDataLocal struct containing the same stored property as the Core Data class
- - Created PersonalTodayVitDlocal struct containing the same stored property as the Core Data class

### CODE REVIEW

File: RayD/RayD/Models/PersonalDataLocal.swift [hunk 1]
```diff
@@ -0,0 +1,17 @@
+//
+//  PersonalDataModel.swift
+//  RayD
+//
+//  Created by Antonio Palomba on 29/05/23.
+//
+
+import Foundation
+
+struct PersonalDataLocal {
```
Comment (yukim17) [inline RayD/RayD/Models/PersonalDataLocal.swift:10]: I would suggest to name it PersonalData**Model**, local seems a little bit weird

File: RayD/RayD/Models/PersonalTodayVitDLocal.swift [hunk 2]
```diff
@@ -0,0 +1,14 @@
+//
+//  PersonalTodayVitDlocal.swift
+//  RayD
+//
+//  Created by Antonio Palomba on 29/05/23.
+//
+
+import Foundation
+
+struct PersonalTodayVitDlocal {
```
Comment (yukim17) [inline RayD/RayD/Models/PersonalTodayVitDLocal.swift:10]: Here's the same about local

File: RayD/PersonalData+CoreDataClass.swift [hunk 3]
```diff
@@ -0,0 +1,15 @@
+//
+//  PersonalData+CoreDataClass.swift
+//  RayD
+//
+//  Created by Antonio Palomba on 29/05/23.
+//
+//
+
+import Foundation
```
Comment (yukim17) [inline RayD/PersonalData+CoreDataClass.swift:9]: It's better to move these classes to CoreData folder

**Review by yukim17 — CHANGES_REQUESTED** [review]

File: RayD/RayD/Models/PersonalDataLocal.swift (hunk 1 above)
Comment (palant-dev) [reply RayD/RayD/Models/PersonalDataLocal.swift:10]: Done!

**Review by slavny11 — APPROVED** [review]

File: RayD/PersonalData+CoreDataClass.swift (hunk 3 above)
Comment (palant-dev) [reply RayD/PersonalData+CoreDataClass.swift]: Done, the files were already there but actually core data was mistakenly addressing the file to the project root folder

**Review by yukim17 — APPROVED** [review]

**Pull request merged**